File: myblog-server/src/main/java/com/site/blog/service/impl/CommentServiceImpl.java
package com.site.blog.service.impl;

import com.site.blog.service.CommentService;
import com.site.blog.model.entity.Comment;
import com.site.blog.mapper.CommentMapper;
import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 评论信息表 服务实现类
 * </p>
 *
 */
@Service
public class CommentServiceImpl extends ServiceImpl<CommentMapper, Comment> implements CommentService {

}

File: blog-sever/server-kt/src/main/kotlin/com/site/blog/service/impl/CommentServiceImpl.kt
package com.site.blog.service.impl

import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl
import com.site.blog.mapper.CommentMapper
import com.site.blog.model.entity.Comment
import com.site.blog.service.CommentService
import org.springframework.stereotype.Service

/**
 *
 *
 * 评论信息表 服务实现类
 *
 *
 */
@Service
class CommentServiceImpl : ServiceImpl<CommentMapper?, Comment?>(), CommentService